zulipterminal/core.py

```diff
@@ -79,7 +79,7 @@ def __init__(
 
         self._editor: Optional[Any] = None
 
-        self.active_conversation_info: Dict[str, Any] = {}
+        self.active_conversation_info: Set[Any] = set()
```

Comment:
Whatever data structure we use here, we should move to a situation where we are not using `Any`, but rather the specific type being used.

Reply:
Will keep this in mind and change `Any` to `str`